README.md

```diff
@@ -0,0 +1,125 @@
+# Gruntwork Installer
+
+[Gruntwork Script Modules](https://github.com/gruntwork-io/script-modules) is a private repo that contains scripts and
+applications developed by [Gruntwork](http://www.gruntwork.io) for common infrastructure tasks such as setting up
+continuous integration, monitoring, log aggregation, and SSH access. This repo contains provides a script called
+`gruntwork-install` that makes it as easy to install the Gruntwork Script Modules as using apt-get, brew, or yum.
+
+For example, in your Packer and Docker templates, you can use `gruntwork-install` as follows:
+
+```bash
+gruntwork-install --module-name 'vault-ssh-helper' --tag '0.0.3'
+```
+
+## Installing gruntwork-install
+
+```bash
+curl -Ls https://raw.githubusercontent.com/gruntwork-io/gruntwork-installer/master/bootstrap-gruntwork-installer.sh | bash -s --version=0.0.1
+```
+
+Notice the `--version` parameter at the end where you specify which version of `gruntwork-install` to install. See the
+[releases](/releases) page for all available versions.
+
+For paranoid security folks, see [is it safe to pipe URLs into bash?](#is-it-safe-to-pipe-urls-into-bash) below.
+
+## Using gruntwork-install
+
+#### Authentication
+
+Since the [Script Modules](https://github.com/gruntwork-io/script-modules) repo is private, you must set your
+[GitHub access token](https://help.github.com/articles/creating-an-access-token-for-command-line-use/) as the
+environment variable `GITHUB_OAUTH_TOKEN` so `gruntwork-install` can use it to access the repo:
+
+```bash
+export GITHUB_OAUTH_TOKEN="..."
+```
+
+#### Options
+
+Once that environment variable is set, you can run `gruntwork-install` with the following options:
+
+* `--module-name`: Required. The name of the Script Module to install. Can be any folder within the `modules` directory
+  of the [Script Modules Repo](https://github.com/gruntwork-io/script-modules).
+* `--tag`: Required. The version of the Script Module to install. Follows the syntax described at [Fetch Version
+  Constraint Operators](https://github.com/gruntwork-io/fetch#version-constraint-operators).
```

Comment:
I changed this URL to https://github.com/gruntwork-io/fetch#tag-constraint-expressions for more consistent naming. The heading is now called "Tag Constraint Expressions."